runtime/runtime/tests/tiny-contract-rs/rust-toolchain

```diff
@@ -0,0 +1 @@
+nightly-2020-10-08
```

Comment:
This can also be build with stable: 

```diff
diff --git a/runtime/runtime/tests/tiny-contract-rs/res/tiny_contract_rs.wasm b/runtime/runtime/tests/tiny-contract-rs/res/tiny_contract_rs.wasm
index 922086db..dd08fdb1 100755
Binary files a/runtime/runtime/tests/tiny-contract-rs/res/tiny_contract_rs.wasm and b/runtime/runtime/tests/tiny-contract-rs/res/tiny_contract_rs.wasm differ
diff --git a/runtime/runtime/tests/tiny-contract-rs/src/lib.rs b/runtime/runtime/tests/tiny-contract-rs/src/lib.rs
index a4ca3284..e8d642c5 100644
--- a/runtime/runtime/tests/tiny-contract-rs/src/lib.rs
+++ b/runtime/runtime/tests/tiny-contract-rs/src/lib.rs
@@ -1,12 +1,9 @@
 #![no_std]
-#![feature(core_intrinsics)]
 
 #[panic_handler]
 #[no_mangle]
 pub fn panic(_info: &::core::panic::PanicInfo) -> ! {
-    unsafe {
-        ::core::intrinsics::abort();
-    }
+    unsafe { core::arch::wasm32::unreachable() }
 }
 
 use core::mem::size_of;
```